easybuild/easyconfigs/p/ParaView/ParaView-4.4.0-gimkl-2.11.5-mpi.eb

```diff
@@ -27,6 +27,7 @@ dependencies = [
     ('libXext', '1.3.3'),
     ('libX11', '1.6.3'),
     ('zlib', '1.2.8'),
+    ('HDF5', '1.8.16', '-serial', ('GCC', '4.9.3')),
```

Comment:
gimkl-2.11.5 is based on GCC-4.9.3 so it should work, no?

I think HDF5/1.8.16 based on GCC is more 'reusable' than based gimkl.

I can fix this anyway.